iina/PrefPluginViewController.swift

```diff
@@ -468,10 +468,26 @@ class PrefPluginViewController: NSViewController, PreferenceWindowEmbeddable {
   private func showNewPluginPermissions(_ plugin: JavascriptPlugin) {
     showPermissionsSheet(forPlugin: plugin, previousPlugin: nil) { ok in
       if ok {
-        plugin.normalizePath()
-        JavascriptPlugin.plugins.append(plugin)
-        plugin.enabled = true
-        self.tableView.reloadData()
+        // check whether a duplicate plugin exists, if yes, replace
+        if JavascriptPlugin.plugins.contains { $0.identifier == plugin.identifier } {
```

Comment:
Xcode generates the following warning for this statement:
```text
/Users/low-batt/Documents/builds/pr-tests/iina/iina/PrefPluginViewController.swift:472:46 Trailing closure in this context is confusable with the body of the statement; pass as a parenthesized argument to silence this warning
```

You should see 6 warnings from Xcode, 5 for use of `NSArchiver` and one for `Info.plist`. There is an open PR for addressing the use of  `NSArchiver` that still needs to be reviewed. The `Info.plist` problem needs to be investigated. An Xcode issue is suspected.

IINA is working to correct all Xcode warnings, so this statement needs to be updated to eliminate the warning.